app/users/page.tsx

```diff
@@ -1,133 +1,82 @@
-              )}
-              {/* localprofileなし */}
-              {profile === undefined ? (
-                <td className="px-6 py-4">localprofileなし</td>
-              ) : (
-                // localprofileあり
-                <td className="px-6 py-4">{profile.phoneNumber}</td>
-              )}
-              {/* localprofileなし */}
-              {profile === undefined ? (
-                <td className="px-6 py-4">localprofileなし</td>
-              ) : (
-                // localprofileあり
-                <td className="px-6 py-4">{profile.mail}</td>
-              )}
-
-              <td className="flex justify-end gap-4 px-6 py-4 font-medium">
-                <button type="button" className="text-primary-700">
-                  clearLocalstrage
-                </button>
-              </td>
-            </tr>
-            <tr className="hover:bg-gray-50">
-              <th className="px-6 py-4">
-                <div className="flex h-5 items-center">
-                  <input
-                    type="checkbox"
-                    id="example12"
-                    name="checkGroup1"
-                    className="h-4 w-4 rounded border-gray-300 text-primary-600 shadow-sm disabled:cursor-not-allowed focus:border-primary-300 disabled:text-gray-400 focus:ring focus:ring-primary-200 focus:ring-opacity-50 focus:ring-offset-0"
-                  />
-                </div>
-              </th>
-              <td className="px-6 py-4 font-medium text-gray-900">
-                Helen Howard
-              </td>
-              <td className="px-6 py-4">samplecompany</td>
-              <td className="px-6 py-4">999999</td>
-              <td className="px-6 py-4">999-999-888</td>
-              <td className="px-6 py-4">samplemail@samplemail.com</td>
-              <td className="flex justify-end gap-4 px-6 py-4 font-medium">
-                <button type="button">Delete</button>
-                <button type="button">Edit</button>
-              </td>
+                <td>{user.id}</td>
+                <td>{String(user.createdAt)}</td>
+                <td>{user.name}</td>
+                <td>{user.company}</td>
+                <td>{user.employeeId}</td>
+                <td>{user.telephone}</td>
+                <td>{user.email}</td>
+                <td>{user.image}</td>
+              </tr>
+            ))
+          ) : (
+            <tr className="text-center">
+              <th colSpan={9}>対応待ちデータなし</th>
             </tr>
-          </tbody>
-        </table>
-        <Link href="/" className="btn btn-primary">
-          <HomeIcon className="h-6 w-6" />
-          ホームに戻る
-        </Link>
-      </>
-    )
-  }
+          )}
+        </tbody>
+        <tfoot>
+          <TableIndex />
+        </tfoot>
+      </table>
+    </div>
+  )
+}
+
+function TableIndex() {
+  const indexList = [
+    'ID',
+    '作成日',
+    '氏名',
+    '所属会社',
+    '社員番号',
+    '電話番号',
+    'メールアドレス',
```

Comment:
I saw these similar items at _const CHECKLIST_ in `/app/register/components/profleForm.tsx`.
I think it's fine as is because it doesn't appear so frequently,
how about making a global list and managing at once?

At least, we had better use a same term "メールアドレス" used here or "Eメール" used in the _const CHECKLIST_ .

Reply:
IMO CHECKLIST is different from tableIndex on `created date`.
That's why I redefined.

In addition, I use short term of "Eメール" for smartphone display.
This table will be watched by tablet or PC display.
Then I choose "メールアドレス".